docs/reference/api/docker_remote_api_v1.24.md

```diff
@@ -3965,7 +3965,7 @@ List nodes
 ### Inspect a node
 
 
-`GET /nodes/<id>`
+`GET /nodes/(id or name)`
```

Comment:
Yes, get/delete already support name in v1.12, but the document has not addressed that.